geoh5py/shared/entity_type.py

```diff
@@ -132,3 +132,18 @@ def workspace(self) -> ws.Workspace:
         # which means workspace has been deleted.
         assert workspace is not None
         return workspace
+
+    def copy(self, workspace: ws.Workspace) -> EntityType:
+        """Copy this entity type to another workspace."""
+
+        attributes = {
+            prop: getattr(self, prop)
+            for prop in dir(self)
+            if isinstance(getattr(self.__class__, prop, None), property)
+            and prop not in ["workspace", "uid"]
```

Comment:
Should only ignore the uid if it's in the same workspace, otherwise keep it.
```suggestion
```

Reply:
If we copy an object, don't we want to let the uid being automatically defined?
We do not need to have the same uid, because it's a new object anyway.

Reply:
Not really if the copy is going to a different workspace. Then GA would be able to recognize they are the same